src/components/ObsDetails/ActivityItem.js

```diff
@@ -11,6 +11,7 @@ import { t } from "i18next";
 import _ from "lodash";
 import type { Node } from "react";
 import React from "react";
+import { Divider } from "react-native-paper";
```

Comment:
There is a custom-styled Divider component in SharedComponents.

Reply:
Oh I didn't realize. Thanks